lib/order.rb

```diff
@@ -0,0 +1,43 @@
+require './lib/menu'
+require './lib/confirmation'
+
+class Order
+
+  attr_reader :basket, :total, :menu, :price_count
+
+  def initialize(menu = Menu.new)
+    @menu = menu
+    @basket = []
+    @total = 0
+    @price_count = []
+  end
+
+  def add(dish)
```

Comment:
This method is currently doing two things - how could you refactor it to give it only one responsibility?